ios/browser/api/ads/ads_client_ios.mm

```diff
@@ -45,12 +45,14 @@
   [bridge_ closeNotification:uuid];
 }
 
-void AdsClientIOS::RecordAdEvent(const std::string& ad_type,
-                                 const std::string& confirmation_type,
-                                 const double timestamp) const {
-  [bridge_ recordAdEvent:ad_type
-        confirmationType:confirmation_type
-               timestamp:timestamp];
+void AdsClientIOS::RecordAdEventForId(const std::string& id,
+                                      const std::string& ad_type,
+                                      const std::string& confirmation_type,
+                                      const double timestamp) const {
+  [bridge_ recordAdEventForId:id
```

Comment:
Same for this one, need to update the Obj-C protocol, won't compile

Reply:
Fixed